packages/components/date-input/stories/time-input.stories.tsx

```diff
@@ -192,6 +193,31 @@ export const WithDescription = {
   },
 };
 
+export const WithErrorMessage = {
+  render: Template,
+
+  args: {
+    ...defaultProps,
+    isInvalid: true,
+    errorMessage: "Please enter a valid time",
+  },
+};
+
+export const WithErrorMessageFunction = {
+  render: Template,
+
+  args: {
+    ...defaultProps,
+    isInvalid: true,
+    errorMessage: "Please enter a valid time",
+  },
+  errorMessage: (value: ValidationResult) => {
```

Comment:
may double check if two `errorMessage` are required or not

Reply:
Thanks I missed it.